coverage/docs/design.md

```diff
@@ -0,0 +1,139 @@
+#Overview
+This code coverage tool has two major features.
+1. In post-submit workflow (when a pull request is merged), it runs code coverage and generates the 
```

Comment:
Again I would omit _all_ mention of the jobs here... the tool does not do GCS uploads. Prow does. This section should just say that the tool can run on source code and emit XML coverage and HTML coverage report